.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
because the app logic relies on git history; `fetch-depth: 0` means fetch the entire history, not just a single commit